torch_em/transform/generic.py

```diff
@@ -73,19 +73,26 @@ def __call__(self, *inputs):
 
 
 class ResizeInputs:
-    def __init__(self, target_shape, is_label=False):
+    def __init__(self, target_shape, is_label=False, is_rgb=False):
         self.target_shape = target_shape
         self.is_label = is_label
+        self.is_rgb = is_rgb
 
     def __call__(self, inputs):
         if self.is_label:  # kwargs needed for int data
             kwargs = {"order": 0,  "anti_aliasing": False}
         else:  # we use the default settings for float data
             kwargs = {}
 
+        if self.is_rgb:
+            assert inputs.ndim == 3 and inputs.shape[0] == 3
+            patch_shape = (3, *self.target_shape)
+        else:
+            patch_shape = self.target_shape
+
```

Comment:
@constantinpape I chose this scheme to handle resizing with rgb images. Could you double-check the logic here? (I tested with this dataset, and seems to works fine)

Reply:
Yes, this looks good to me.